sql/moz-fx-data-shared-prod/firefox_desktop/baseline_active_users/view.sql

```diff
@@ -2,62 +2,79 @@ CREATE OR REPLACE VIEW
       THEN CONCAT('Firefox Desktop', ' ', isp)
-    WHEN LOWER(IFNULL(distribution_id, '')) = 'mozillaonline'
+    WHEN distribution_id = 'MozillaOnline'
       THEN CONCAT('Firefox Desktop', ' ', distribution_id)
     ELSE 'Firefox Desktop'
   END AS app_name,
+  app_display_version AS app_version,
+  normalized_channel AS channel,
+  normalized_os AS os,
+  COALESCE(
+    `mozfun.norm.windows_version_info`(normalized_os, normalized_os_version, windows_build_number),
+    normalized_os_version
+  ) AS os_version_build,
+  COALESCE(
+    CAST(NULLIF(SPLIT(normalized_os_version, ".")[SAFE_OFFSET(0)], "") AS INTEGER),
+    0
+  ) AS os_version_major,
+  COALESCE(
+    CAST(NULLIF(SPLIT(normalized_os_version, ".")[SAFE_OFFSET(1)], "") AS INTEGER),
+    0
+  ) AS os_version_minor,
   CASE
-    WHEN BIT_COUNT(days_active_bits)
+    WHEN BIT_COUNT(days_desktop_active_bits)
       BETWEEN 1
       AND 6
       THEN 'infrequent_user'
-    WHEN BIT_COUNT(days_active_bits)
+    WHEN BIT_COUNT(days_desktop_active_bits)
       BETWEEN 7
       AND 13
       THEN 'casual_user'
-    WHEN BIT_COUNT(days_active_bits)
+    WHEN BIT_COUNT(days_desktop_active_bits)
       BETWEEN 14
       AND 20
       THEN 'regular_user'
-    WHEN BIT_COUNT(days_active_bits) >= 21
+    WHEN BIT_COUNT(days_desktop_active_bits) >= 21
       THEN 'core_user'
     ELSE 'other'
-  END AS activity_segment,
+  END AS segment_dau,
   CASE
-    WHEN BIT_COUNT(days_desktop_active_bits)
+    WHEN BIT_COUNT(days_seen_bits)
       BETWEEN 1
       AND 6
       THEN 'infrequent_user'
-    WHEN BIT_COUNT(days_desktop_active_bits)
+    WHEN BIT_COUNT(days_seen_bits)
       BETWEEN 7
       AND 13
       THEN 'casual_user'
-    WHEN BIT_COUNT(days_desktop_active_bits)
+    WHEN BIT_COUNT(days_seen_bits)
       BETWEEN 14
       AND 20
       THEN 'regular_user'
-    WHEN BIT_COUNT(days_desktop_active_bits) >= 21
+    WHEN BIT_COUNT(days_seen_bits) >= 21
       THEN 'core_user'
     ELSE 'other'
-  END AS segment_dau,
-  IFNULL(mozfun.bits28.days_since_seen(days_active_bits) = 0, FALSE) AS is_dau_duration,
-  IFNULL(mozfun.bits28.days_since_seen(days_active_bits) < 7, FALSE) AS is_wau_duration,
-  IFNULL(mozfun.bits28.days_since_seen(days_active_bits) < 28, FALSE) AS is_mau_duration,
-  IFNULL(mozfun.bits28.days_since_seen(days_desktop_active_bits) = 0, FALSE) AS is_dau,
-  IFNULL(mozfun.bits28.days_since_seen(days_desktop_active_bits) < 7, FALSE) AS is_wau,
-  IFNULL(mozfun.bits28.days_since_seen(days_desktop_active_bits) < 28, FALSE) AS is_mau,
-  IFNULL(mozfun.bits28.days_since_seen(days_seen_bits) = 0, FALSE) AS is_daily_user,
-  IFNULL(mozfun.bits28.days_since_seen(days_seen_bits) < 7, FALSE) AS is_weekly_user,
-  IFNULL(mozfun.bits28.days_since_seen(days_seen_bits) < 28, FALSE) AS is_monthly_user,
-  IF(
-    LOWER(IFNULL(isp, '')) <> "browserstack"
-    AND LOWER(IFNULL(distribution_id, '')) <> "mozillaonline",
-    TRUE,
-    FALSE
-  ) AS is_included_kpi
+  END AS activity_segment,
```

Comment:
@irrationalagent I'm not sure this naming is really appropriate here, active_segment is based off days_seen_bits which can cause some confusion. Do we need both segments (segment_dau and activity_segment)?

Reply:
Good call out. In our current logic for [desktop_active_users](https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry/desktop_active_users/view.sql#L9-L24) activity segment is based only on days_active_bits, which is analogous to days_desktop_active_bits here. So, I suppose we should mirror that behavior and have a single `activity_segment` column based on `days_desktop_active_bits`. 

If people need to segment usage history based on pings rather than DAU, then days_seen_bits is still available for them to do that manually.

Reply:
@irrationalagent ok, I updated the query. dau_segment was removed and activity_segment is using `days_desktop_active_bits` now.